ohtu2021-kvantti/WebMark
# Several molecules

## Summary

Major revision. Algorithm and molecule is now seperated. Algorithm is not tight to single molecule. Version of algorithm can be tested with different molecules. Comparison can be made only by metrics produced by same molecule.  

## How to test

Makemigrations and migrate needed.

Insert molecules, algorithms, several version of algorithms, several measurements with different molecules but same versions. 

Check that inserted data can be viewed in algorithm details view and comparison work as expected.

Probably there is some issues which must be fixed before merge
